### PR TITLE
Feature/us6550 utility classes

### PR DESCRIPTION
**Pull request by zarahzachz**
@tcmacdonald I cleaned up a lot of redundant code, using Sass mixins to accomplish a lot of the "mobile-specific classes".

Corresponds with crdschurch/crds-styleguide#49

### CODE REVIEW

File: assets/stylesheets/base/_utilities.scss [hunk 1]
```diff
@@ -398,97 +156,67 @@ a.underline,
...
+        }
+      }
     }
   }
-  .sm-pull-left {
-    float: left;
-  }
 }
 
+@include floats;
```
Comment (tcmacdonald) [inline assets/stylesheets/base/_utilities.scss]: You've got a couple mixins here that are only used in one place. Does that make sense? Should mixins belong here and not in the mixins module?

**Comment by zarahzachz** [comment]
@tcmacdonald I've cleaned up the mixins, pulling out the `@mixin` wrapper on the single-use cases, and added comments explaining my thought-process on the code.

**Comment by zarahzachz** [comment]
@tcmacdonald I added some more instruction in the Sass file on what types of classes would be rendered - giving more instruction to the developer.

Ready for review.

**Comment by zarahzachz** [comment]
@tcmacdonald I refactored the utility classes and stripped out the Sassy logic.

Ready for review.

**Pull request merged**